### ISSUE TITLE
Code coverage badges

### ISSUE THREAD

**Issue by martinamorris** [statnet/statnet#2]
@krivit do you have any objections to adding this badge to all of the statnet  repositories that are ultimately destined for CRAN?

if not, i'm happy to add them as i'm working through the revdep stuff, but i'd like to make sure i'm adding the right thing to the travis.yml file.  from `ergm.ego` you have:

```
after_success:
- Rscript -e 'covr::codecov(type="all")'
```

is that right?  anything else needed?

**Comment by krivit** [comment]
I think so. It's been a while.

**Comment by mbojan** [comment]
Its been a while, but didn't we have one on ergm and later removed it because CI became veeery slow?

**Comment by martinamorris** [comment]
@mbojan would be worth re-checking.   do you mean that it slows down the travis-ci process?

**Comment by mbojan** [comment]
Yes, CI jobs became very slow. I need to look up the history though.

**Comment by smjenness** [comment]
I had it turned on for EpiModel for some time, but then decided to turn it off because it essentially doubled the time on Travis CI. I believe because Travis will first run R CMD Check, and then if that passes, all the examples/unit tests again with codecov.
